src/pages/dar_application/ResearcherInfo.js

```diff
@@ -292,14 +311,17 @@ export default function ResearcherInfo(props) {
               ])
             ]),
             div({className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'}, [
-              input({
-                type: 'text',
-                defaultValue: signingOfficial,
-                name: 'signingOfficial',
-                required: true,
-                className: isEmpty(signingOfficial) && showValidationMessages ? 'form-control required-field-error' : 'form-control',
-                onBlur: (e) => formFieldChange({name: 'signingOfficial', value: e.target.value})
-              }),
+              <Creatable
```

Comment:
Unrelated to the scope of this PR. We are already using `AsyncSelect` for all of our selection menus. Adding `Creatable` here seems redundant.

Reply:
Update ... ignore this comment - did not realize they are in the same library.